src/antsibull_core/utils/hashing.py

```diff
@@ -8,14 +8,32 @@
 from __future__ import annotations
 
 import hashlib
+import typing as t
+from collections.abc import Mapping
 
 import aiofiles
 
 from .. import app_context
 
 
+class _AlgorithmData(t.NamedTuple):
+    name: str
+    algorithm: str
+    kwargs: dict
+
+
+_PREFERRED_HASHES = [
+    # https://pypi.org/help/#verify-hashes, https://github.com/pypi/warehouse/issues/9628
+    _AlgorithmData("sha256", "sha256", {}),
+    _AlgorithmData("blake2b_256", "blake2b", {"digest_size": 32}),
+]
```

Comment:
```suggestion
_PREFERRED_HASHES: tuple[__AlgorithmData, ...] = (
    # https://pypi.org/help/#verify-hashes, https://github.com/pypi/warehouse/issues/9628
    _AlgorithmData("sha256", "sha256", {}),
    _AlgorithmData("blake2b_256", "blake2b", {"digest_size": 32}),
)
```

I prefer tuples for module level constants, and this could use a type annotation.